monGARS/config.py

```
@@ -90,6 +90,33 @@ class Settings(BaseSettings):
         "true",
         "1",
     )
+    style_base_model: str = Field(
+        default="hf-internal-testing/tiny-random-gpt2",
+        validation_alias="STYLE_BASE_MODEL",
+    )
+    style_adapter_dir: str = Field(
+        default=os.getenv("STYLE_ADAPTER_DIR", "/tmp/mongars_style"),
```

Ruff (0.13.1)

Probable insecure usage of temporary file or directory: "/tmp/mongars_style"

(S108)